features/bevragen/persoon/verblijfplaats/onbekend/protocollering-antwoord.feature

```diff
@@ -0,0 +1,34 @@
+#language: nl
+
+Functionaliteit: protocollering van het antwoord voor verblijfplaats onbekend
+
```

Comment:
Moet in dit bestand ook nog een Rule en bijbehorend scenario worden gedefinieerd voor het vastleggen van geleverde velden? Die is immers niet gelijk aan die van de gevraagde velden.